src/components/custom/_dtrading.tsx

```diff
@@ -209,46 +207,25 @@ const query = graphql`
         }
     }
 `
-const DTrading = ({
-    trading,
-    reverse,
-    two_title,
-    max_width,
-    max_height,
-    spacing,
-}: DTradingProps) => {
+
+const DTrading = ({ trading, reverse, max_width, max_height, spacing }: DTradingProps) => {
     const data = useStaticQuery(query)
+
     return (
         <StyledSection $spacing={spacing}>
             <Container direction="column">
-                {trading.map((item, index) => {
+                {trading.map(({ image_alt, image_name, subtitle, title }, index) => {
                     const is_even = reverse ? (index + 1) % 2 : index % 2
+
                     return (
-                        <Row
-                            flex_direction={!is_even ? 'row' : 'row-reverse'}
-                            key={
-                                typeof item.title === 'string'
-                                    ? item.title
-                                    : item.title.props.translate_text
-                            }
-                        >
+                        <Row flex_direction={!is_even ? 'row' : 'row-reverse'} key={title}>
```

Comment:
```suggestion
                        <Row flex_direction={is_even ?  'row-reverse':'row' } key={title}>
```
just for more readability